README.md

```diff
@@ -70,6 +70,16 @@ In this case, please navigate the tags in the repository
 (e.g. [v0.4.5](https://github.com/fairlearn/fairlearn/tree/v0.4.5))
 to locate the appropriate version of the notebook.
 
+## Resources
+
+- [fairlearn.github.io](https://fairlearn.github.io/)
+- [FairLearn Whitepaper](https://www.microsoft.com/en-us/research/uploads/prod/2020/05/Fairlearn_whitepaper.pdf)
+- [FairLearn Documentation](https://fairlearn.github.io/auto_examples/notebooks/index.html)
+- [Fairlearn Examples](https://fairlearn.github.io/auto_examples/notebooks/index.html)
+- [Building fairer AI Systems with Fairlearn on Channel 9](https://channel9.msdn.com/Shows/AI-Show/Building-fairer-AI-Systems-with-Fairlearn)
```

Comment:
@Rishit-dagli thanks for adding resources! We've deliberately made the decision not to add these to the open source project since Fairlearn is meant to be separate from Microsoft. All of these are Microsoft branded and at least partially refer to the additional capabilities in AzureML. 

For the remaining links I think it might make sense to link to the website somewhere at the start of the README, then people can find the rest through the website. The whitepaper is an interesting case. @MiroDudik I believe you wanted to have a slightly adjusted version linked from Fairlearn?

Reply:
Actually now I'm seeing the website is already linked from the README, line 7. So it comes down to the whitepaper only.

Reply:
Sounds great, maybe just the whitepaper somewhere in the documentation?

Reply:
The "Further Resources" page, perhaps:
https://github.com/fairlearn/fairlearn/blob/master/docs/user_guide/further_resources.rst

Reply:
@riedgar-ms That seems like a great idea, I will make that edit